debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -1527,6 +1527,7 @@ C|J9_BCLOOP_J2I_TRANSITION
 C|J9_BCLOOP_J2I_VIRTUAL
 C|J9_BCLOOP_JUMP_BYTECODE_PROTOTYPE
 C|J9_BCLOOP_LOAD_PRESERVED_AND_BRANCH
+C|J9_BCLOOP_N2I_TRANSITION
```

Comment:
No entries should be added to this file; see https://github.com/eclipse-openj9/openj9/pull/15611#discussion_r970106417 and #15883.